src/Integration/Connection/ConnectionWorkflow.cs

```diff
@@ -49,17 +49,13 @@ internal class ConnectionWorkflow
         private readonly IHost host;
         private readonly ICommand parentCommand;
         private readonly ICredentialStoreService credentialStore;
-        private readonly ITestProjectRegexSetter testProjectRegexSetter;
 
         public ConnectionWorkflow(IHost host, ICommand parentCommand)
         {
             this.host = host ?? throw new ArgumentNullException(nameof(host));
             this.parentCommand = parentCommand ?? throw new ArgumentNullException(nameof(parentCommand));
 
-            this.credentialStore = this.host.GetMefService<ICredentialStoreService>();
-
-            this.testProjectRegexSetter = this.host.GetService<ITestProjectRegexSetter>();
```

Comment:
Previously, the connection workflow had a step that downloaded server settings so it could pass one of them to `ITestProjectRegexSetter`, which was then later used to filter out test projects.

The new unintrusive binding doesn't do any filtering since no project files are changed 
-> the `ITestProjectRegexSetter` doesn't need to be set 
-> the server settings don't need to be downloaded.